go/ct/common/hash256_test.go

```diff
@@ -0,0 +1,115 @@
+//
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use
+// of this software will be governed by the GNU Lesser General Public Licence v3
+//
+
+package common
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/Fantom-foundation/Tosca/go/vm"
+)
+
+func TestHash256_EqualWhenContainingSameContent(t *testing.T) {
```

Comment:
sorry, both filename and the unit tests names are out of sync now with the type name.
(I really don't like to write 256 in the type name. But I guess this is my problem ;) )